src/components/storefront/CartItem/CartItem.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Heading, Flex, Image, useNumberInput, IconButton, VStack, Spacer } from '@chakra-ui/react';
+import QuantityPicker from '../ProductDetails/QuantityPicker';
+import { CloseIcon } from '@chakra-ui/icons';
+
+const CartItem = ({ product }) => {
+  const { images, title, variants } = product;
+  const productSku = variants[0].sku;
+  const productImage = images[0].src;
+  const productPrice = variants[0].price;
+  const productQuantity = 1; // Fetch quantity from checkout line items. This should also be a state (to update price & total)
+
+  const quantityPickerProps = useNumberInput({
+    step: 1,
+    defaultValue: productQuantity,
+    min: 1,
+    max: 999,
+  });
+
+  const removeFromCart = () => {
+    alert('To be implemented');
+  };
+
+  return (
+    <Flex>
+      <Flex direction="column">
+        <Image boxSize="200" objectFit="cover" border="1px solid black" src={productImage} alt={title} />
+      </Flex>
```

Comment:
at a glance it doesn't seem necessary to wrap the image in a flexbox but I may be wrong

Reply:
The reason I wrapped it in a container is so that all of the images would be the same width/height, this is without the flexbox
![image](https://user-images.githubusercontent.com/24748963/113785828-e2c31b00-9705-11eb-9dc4-7252f0cc0dbe.png)

Reply:
the reason I added a flexbox is so that it scales a bit nicer, e.g. this is with a smaller window size without the flexbox
![image](https://user-images.githubusercontent.com/24748963/113785923-1dc54e80-9706-11eb-94d8-ddfea2203583.png)

and this is what it looks like with a flexbox
![image](https://user-images.githubusercontent.com/24748963/113785976-32094b80-9706-11eb-878d-b17b42d0f29d.png)

not sure with one is more desired though

Reply:
Ah ok I got it, in that case it's fine — I'll do all responsive styling in one big PR i think

it would probably be ideal to force all images the a square of the same size or something but it really isn't a big deal at all, I wouldn't stress it